### ISSUE TITLE
NVDA does not read a table created with DIVs and table roles

### ISSUE THREAD

**Issue by DavidMacDonald** [nvaccess/nvda#5479]
See this page
http://davidmacd.com/test/grid.html

**Comment by jcsteh** [comment]
Can you explain the problem you're having exactly? For me, in Firefox, 
NVDA announces this as a table, announces row and column coordinates and 
table navigation commands move between the rows and columns as expected.

**Comment by DavidMacDonald** [comment]
The developers of NVDA tweeted me and said they would fix it. So that is probably what happened.

**Comment by jcsteh** [comment]
> The developers of NVDA tweeted me and said they would fix it. So that is probably what happened.

I'm confused. I *am* a developer of NVDA. Do you mean the developers of Firefox or something else?

**Comment by ehollig** [comment]
@DavidMacDonald could you provide more information on the issue you are having? Otherwise, closing because of lack of information.
Testing performed:
I opened the above URL in Firefox 54 and was able to navigate the table using table movement commands. NVDA did announce the headers of each column as I moved between the columns.
